src/test/java/selenium/MultiselectTest.java

```diff
@@ -0,0 +1,48 @@
+package selenium;
+
+import lombok.extern.slf4j.Slf4j;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import testbase.TestBase;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import static constants.ByConstants.MULTISELECT;
+import static constants.ByConstants.MULTISELECT_OPTION;
+import static constants.TestConstants.MULTISELECT_TEST_URL;
+
+@Slf4j
+public class MultiselectTest extends TestBase {
+
+    @Test
+    public void testMultiselect() {
```

Comment:
test multiselect what? we need to state the goal of testing by meaningful name (like testcase name)

Reply:
Done!